src/pages/events/consume-events.md

```diff
@@ -0,0 +1,101 @@
+
+Below is a JavaScript code sample that could be used to create a runtime action for consuming `observer.sales_order_save_after` events. 
+
+```js
+const { Core } = require('@adobe/aio-sdk')
+const { errorResponse } = require('../utils')
+const { getCommerceOauthClient } = require('../oauth1a')
+const { sendOrderToErpSystem } = require('../erp')
+  
+async function main (params) {
+  const logger = Core.Logger('main', { level: params.LOG_LEVEL || 'info' })
+  
+  try {
+    event_payload = params.data.value
+    if (!event_payload.hasOwnProperty('extension_attributes')) {
+      // Fetch extension attributes for order and add to order event payload
+      const oauth = getCommerceOauthClient(
+        {
+          url: params.COMMERCE_BASE_URL,
+          consumerKey: params.COMMERCE_CONSUMER_KEY,
+          consumerSecret: params.COMMERCE_CONSUMER_SECRET,
+          accessToken: params.COMMERCE_ACCESS_TOKEN,
+          accessTokenSecret: params.COMMERCE_ACCESS_TOKEN_SECRET
+        },
+        logger
+      )
+      const content = await oauth.get('orders/' + event_payload.entity_id)
+      event_payload.extension_attributes = content.extension_attributes
+    }
+
+    // Send event data to a third-party ERP system
+    sendOrderToErpSystem(event_payload)
+      
+    return {
+      statusCode: 200,
+      body: event_payload
+    }
+  } catch (error) {
+    // log any server errors
+    logger.error(error)
+    // return with 500
+    return errorResponse(500, 'server error', logger)
+  }
+}
+  
+exports.main = main
+```
+
+The `main` actions first accesses the payload for a received `observer.sales_order_save_after` event:
+```js
+event_payload = params.data.value
+```
+
+The event payload for this event may not contain the saved order's extension attributes. In that case, the extension attributes can be fetched for the specific order using a Commerce API call:
+```js
+  const oauth = getCommerceOauthClient(
+    {
+      url: params.COMMERCE_BASE_URL,
+      consumerKey: params.COMMERCE_CONSUMER_KEY,
+      consumerSecret: params.COMMERCE_CONSUMER_SECRET,
+      accessToken: params.COMMERCE_ACCESS_TOKEN,
+      accessTokenSecret: params.COMMERCE_ACCESS_TOKEN_SECRET
+    },
+    logger
+  )
+  const content = await oauth.get('orders/' + event_payload.entity_id)
+```
+In this example, functions from the `oauth1a` module, as defined in the [adobe-commerce-samples repo](https://github.com/adobe/adobe-commerce-samples/blob/main/sample-extension/actions/oauth1a.js), are used. 
```

Comment:
TODO: this may move as part of https://github.com/adobe/adobe-commerce-samples/pull/9